tests/integration/targets/role_check_local_time/tasks/main.yml

```diff
@@ -0,0 +1,38 @@
+---
+- environment:
+    SC_HOST: "{{ sc_host }}"
+    SC_USERNAME: "{{ sc_username }}"
+    SC_PASSWORD: "{{ sc_password }}"
+    SC_TIMEOUT: "{{ sc_timeout }}"
+
+  block:
+    - name: Check that local time meets required time interval
+      ansible.builtin.include_role:
+        name: scale_computing.hypercore.check_local_time
+      vars:
+        time_zone: "{{ ansible_date_time.tz }}"
+        time_interval: "{{ ansible_date_time.hour }}:00-{{ ansible_date_time.hour }}:59"
```

Comment:
It would be easier to test if current time would be additional input variable. It could be optional, with default value == ansible_date_time.

The current test - it can fail when run near x:00 or x:59 or x:01. Mention this in comment.

Test also demonstrates usage. It would be easier to use if role would just fail if current time is not within `time_interval`. Otherwise, every user will need to copy-paste 2 or 3 tasks - easy to add also a typo (variable names).

Reply:
@justinc1 Role does fail if current time is not within time_interval, that's why apply: ignore errors is used in integration test. But still I needed something to assert the error (so that random error is not passed through) - now I am using ansible fact. But when user will use this role, these additional tasks will not be needed.

Reply:
@justinc1 in case you missed the comment